src/test/java/org/prebid/server/analytics/reporter/greenbids/GreenbidsAnalyticsReporterTest.java

```diff
@@ -540,16 +646,32 @@ private static CommonMessage givenCommonMessage(
                 .build();
     }
 
-    private static List<GreenbidsBids> givenGreenbidsBids() {
+    private static List<GreenbidsBid> givenGreenbidsBids() {
```

Comment:
`expectedGreenbidBids` makes more sense

please also rename other methods, that build the expected result, like `givenCommonMessage`